spec/bolt/project_spec.rb

```diff
@@ -78,9 +78,9 @@
     describe "with namespaced project names" do
       let(:config) { { 'name' => 'puppetlabs-foo' } }
 
-      it "strips namespace and hyphen" do
-        project = Bolt::Project.new(config, pwd)
-        expect(project.name).to eq('foo')
+      it "raises an error" do
+        expect { Bolt::Project.new(config, pwd).validate }
+          .to raise_error(/Invalid project name 'puppetlabs-foo' in bolt-project.yaml/)
```

Comment:
I think we also need to add `name:` to the bolt-project.yaml files in `spec/fixtures/projects`.

Reply:
Whoops apparently I forgot to commit those files. Classic.